frontend/src/container/AnomalyAlertEvaluationView/tooltipPlugin.ts

```diff
@@ -17,7 +19,7 @@ const tooltipPlugin = (
 			return value.toFixed(3);
 		}
 		if (value instanceof Date) {
-			return value.toLocaleString();
+			return dayjs(value).tz(timezone).format('MM/DD/YYYY, h:mm:ss A');
```

Comment:
earlier this was string coming from consumer, you had a format, just recheck if this is expected, and you are not overriding anything

Reply:
@SagarRajput-7, this is the tooltip plugin for anomaly alert. No, we aren't overriding anything, this is the only change that we have made, i.e., introduced timezone-adjusted timestamp formatting.